src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--block">
+      <ul class="stars stars--0">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+      </ul>
+      <ul class="stars stars--1">
+        <li class="stars__star stars__star_active"></li>
```

Comment:
You should have only one class here and in the same cases where you use additional class for active star. 
The number of active stars must be specified in the modifier, which is defined in the container for each row of stars